config/initializers/job_configurations.rb

```diff
@@ -217,6 +217,12 @@
         class: 'ThreatMetrixJsVerificationJob',
         cron: cron_1h,
       },
+      # Batch up IRS Attempts API events
+      irs_attempt_events_aggregator: {
+        class: 'IrsAttemptsEventsBatchJob',
+        cron: cron_1h,
+        args: -> { [timestamp: Time.zone.now - 1.hour] },
```

Comment:
this is an array of a hash, let's add in braces to make that clearer?
```suggestion
        args: -> { [ { timestamp: Time.zone.now - 1.hour } ] },
```

Reply:
Done